### PR TITLE
Feature/dialog event improvements

### PR DESCRIPTION
**Pull request by serjKim**
Fix comments https://github.com/SoftwareCountry/arcadian-assistant/pull/63
and others.

### CODE REVIEW

File: mobile/src/calendar/calendar-actions-button-group.tsx [hunk 1]
```diff
@@ -74,7 +73,10 @@ const mapStateToProps = (state: AppState) => ({
 });
 
 const mapDispatchToProps = (dispatch: Dispatch<CalendarActions>) => ({
-    editSickLeave: () => { dispatch(editSickLeave()); }
+    sickLeaveActions: {
+        claim: () => { dispatch(claimSickLeave()); },
```
Comment (santee) [inline mobile/src/calendar/calendar-actions-button-group.tsx]: are {} really needed?

File: mobile/src/reducers/calendar/calendar-events.reducer.ts [hunk 2]
```diff
@@ -66,27 +67,47 @@ export const calendarEventsReducer: Reducer<CalendarEventsState> = (state = init
                 ...state,
                 weeks: weeks
             };
+        case 'CLAIM-SICK-LEAVE':
+            const claimDialog: DialogActiveState = {
```
Comment (santee) [inline mobile/src/reducers/calendar/calendar-events.reducer.ts]: i think this configuration things can be moved to separate constant in a separate file? To separate constants definitely.

File: mobile/src/calendar/calendar-actions-button-group.tsx [hunk 3]
```diff
@@ -35,37 +39,32 @@ export class ActionsButtonGroupImpl extends Component<ActionButtonGroupProps & A
 
                 <CalendarActionButtonSeparator />
 
-                <SickLeaveActionButton interval={sickleave} onPress={this.onSickLeaveAction} />
+                <SickLeaveActionButton interval={sickleave} {...this.props.sickLeaveActions} />
```
[the remark this answers is not in the document]
Comment (serjKim) [reply mobile/src/calendar/calendar-actions-button-group.tsx]: yes

File: mobile/src/calendar/calendar-actions-button-group.tsx (hunk 1 above)
Comment (serjKim) [reply mobile/src/calendar/calendar-actions-button-group.tsx]: I think it looks convenient with spread operator

**Pull request merged**